github/ci/prow/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```
@@ -1,5 +1,35 @@
 postsubmits:
   kubevirt/project-infra:
+    - name: publish-rehearse-image
+      always_run: false
+      run_if_changed: "external-plugins/rehearse/.*"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-project-infra-kubevirtci-docker-credential: "true"
+      spec:
+        nodeSelector:
+          type: vm
+        containers:
+          - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - >-
```

I think this will not work.
```suggestion
              - >
```

---

Why not? with `>` we're adding a newline (`\n`) at the end and with `>-` we're not.

---

TIL sth new :) Thanks!